GoogleForCreators/web-stories-wp
# Mark videos in templates as having no sound

## Context



## Summary

Currently, the pre-publish checklist shows a message for a video with missing captions.

Videos in our templates don't have any sound though, so the message is very misleading in that case.

This PR requires #8340 to be merge first. 

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Go to Stories > export templates. 
2. Create a story for the "Kitchen stories" template. 
3. Save story. 
4. Ensure that their are no accessibility prepublish items regarding captions for videos embedded in story. 

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #7885
